### ISSUE TITLE
Format notification missing in packaged version

### ISSUE THREAD

**Issue by camilleanne** [mapbox/mapbox-studio-classic#805]
This isn't showing up when the entered name doesn't match the pattern:

![screen shot 2014-09-09 at 1 16 22 pm](https://cloud.githubusercontent.com/assets/3952537/4208326/3b7cc960-385f-11e4-847a-f52e076a5753.png)

cc @yhahn

**Comment by yhahn** [comment]
Yeah, this is the HTML5 validation warning coming from Chrome/Firefox/etc. Guessing atom-shell/chromium upstream hasn't implemented/exposed this yet.

Next task here would be poking around upstream and seeing what the plan is for this (we're also not fully up to date on atom-shell so maybe we start by upgrading).

**Comment by camilleanne** [comment]
It's not in [0.16.2](https://github.com/atom/atom-shell/releases) yet.

**Comment by springmeyer** [comment]
@camilleanne - In v0.2.0 we're now upgraded to atom-shell v0.19.2 (https://github.com/atom/atom-shell/releases/tag/v0.19.2) - want to test this again?

**Comment by camilleanne** [comment]
@springmeyer :+1:  I'll take a look.

**Comment by camilleanne** [comment]
Unfortunately there is no sign of it in 0.2.0 :/

**Comment by camilleanne** [comment]
Still nothing as of [v0.20.7](https://github.com/atom/atom-shell/releases/tag/v0.20.7) of atom-shell.

**Comment by springmeyer** [comment]
@camilleanne - what about creating a ticket at atom-shell tracker?

**Comment by camilleanne** [comment]
https://github.com/atom/atom-shell/issues/1043

**Comment by camilleanne** [comment]
Looks like we're going this one on our own:
https://github.com/atom/atom-shell/issues/1043#issuecomment-70952338

**Comment by springmeyer** [comment]
I think we should remove all html input pattern validation altogether. The regexes for the file browser in particular are quite limiting and we should just keep it simple by showing all files and depending on mapnik-omnivore to throw if they are not valid to be opened - refs #1149.

**Comment by yhahn** [comment]
Hrm, we use html 5 input validation for more things than just file browser filetypes. We'll have to mull on a good approach to this one.

**Comment by camilleanne** [comment]
I agree with @yhahn because of issues like #1123. So we can make [our own notifications](http://developer.telerik.com/featured/building-html5-form-validation-bubble-replacements/) (I've already looked into this, it's not that bad), or we can make the change source interface better.

**Comment by springmeyer** [comment]
@camilleanne @yhahn - any thoughts on what is lowest hanging and feasible to move on here? #1342 reminds me of how nice it would be to fix this so that errors are not silent.
